kernel/src/debug.rs

```diff
@@ -220,7 +220,7 @@ pub struct DebugWriter {
 static mut DEBUG_WRITER: Option<&'static mut DebugWriterWrapper> = None;
 
 pub static mut OUTPUT_BUF: [u8; 64] = [0; 64];
-pub static mut INTERNAL_BUF: [u8; 1024] = [0; 1024];
+pub static mut INTERNAL_BUF: [u8; 1024 - 64] = [0; 1024 - 64];
```

Comment:
Probably would be good to have a comment here so this doesn't get changed back...changes here seem to pop up in PRs from time to time (at least historically).

Reply:
A comment definitely makes sense, I'll add.

One saving grace is that this is only a problem for memory overuse, and so will either _not_ be an issue, or will show up as a linker error at least for HiFive1 board (so Travis will catch it).